src/site/_includes/partials/language-list.njk

```diff
@@ -0,0 +1,19 @@
+<span class="post-signpost__title">
+  {% set languages = page | supportedlanguages(lang) %}
+  {% set numberOfLanguages = languages.length %}
+
+  {% if (numberOfLanguages) %}
+    {{'i18n.post.available_in' | i18n(locale)}}:
+    {% for language in languages %}
+      {% set lang = language[0] %}
+      {% set langHref = language[1] %}
+      {% set langName = language[2] %}
+
+      <span>
+        <a href={{langHref}} class="post-signpost__link" translate="no" lang={{lang}}>{{langName}}</a>
+        {%- if loop.index !== numberOfLanguages -%}, {%- endif -%}
+        {% if loop.index === (numberOfLanguages - 1) %} and {% endif %}
```

Comment:
I think you don't need `numberOfLanguages` here and the loop can be simplified, as you have `loop.first`, `loop.last` and `loop.length` in Nunjucks. See https://mozilla.github.io/nunjucks/templating.html#:~:text=loop.first%3A%20boolean,loop.last